NPS.ID.PublicApi.DotNet.Client/Connection/Enums/WebSocketClientTarget.cs

```diff
@@ -0,0 +1,8 @@
+namespace NPS.ID.PublicApi.DotNet.Client.Connection.Enums;
+
+public enum WebSocketClientTarget
+{
+    None = 0,
+    Middleware,
```

Comment:
Do these make sense to the clients? Maybe we should rename them to smth like Trading and MarketData?

Reply:
Trading and MarketData or PublicMarketData sounds good for me, Indeed "Middleware" says nothing to our clients

Reply:
Names changed